cloth-diapers/cloth-diapers-basics.md

```diff
@@ -0,0 +1,296 @@
+choices we have made (see below).
+
+## Materials and Absorbency
+
+<img width="664" alt="image" src="https://user-images.githubusercontent.com/4185328/169095781-8c3c8f69-3ae5-4e04-ab39-780f198651a5.png">
+
+Cloth diapers hinge on both their speed of and capacity for absorbency.
+
+The only other thing to mention other than the table above is 'Charcoal Bamboo'.
+This is actually not bamboo at all, it's just a fancy name for microfibre and
+equally unhelpful.
+
+### Absorbency capacity
+The more capacity for absorbency, the more liquid a diaper will hold.
+
+Having said this, every baby is different so whilst some can get away with
+very little absorbency, others are considered 'medium to heavy wetters' and
+will need a lot more absorbency, relying much more heavily on hemp inserts
+(typically used in pocket diapers) or boosters (typically layered into other
+types of diapers to increase absorbency).
+
+### Absorbency speed
+This is important for two reasons:
+1. You need to layer your absorbency so that the layers closer to the baby
+are the ones that absorb liquids the fastest - this will allow that layer to
+absorb the liquids and wick them away to the lower layers which will have more
+storage capacity.
+2. If you have a baby who 'floods'; i.e. releases a large quantity of liquid
+in a short time frame. Here you want the fastest possible absorption to wick
+away the moisture so that you don't get leaks.
+
+## Newborn vs One Size
+There are 3 'sizes' of cloth diapers:
++ **Newborn**: These are extra small diapers that in some cases (not all) also
+fit preemies. They have snaps that allow them to be resized to 2 or 3 different
+sizes as well.
++ **One Size:** Often referred to as ***OS*** diapers, these are the typical
+size of modern cloth diaper that **will accompany a child throughout the
+majority of their diaper journey**. They are marketed as fitting babies from
+around 3-15kg (with a small amount of variation per brand) but usually don't fit
+newborns because the leg holes are designed for slightly bigger babies.
++ **XL:** Nomenclature varies but these are often designated 'XL' or 'Super'
+and are made for older toddlers but sizes vary widely. They could refer to
+diapers that are 9 - 18kgs for example or even diapers that are 15 - 20kgs, so
+it's important to review the stipulations of fit from the individual brand.
+
+**One size (OS) are by _far_ the most common**, with many people deciding to
+wait until their babies fit into this size before they start their cloth diaper
+journey.
+
+There _are_ 'newborn hacks' of folds that allow you to use OS diapers on smaller
+babies but these are greatly dependent on the baby's body and on the diaper
+brands.
+
+# Our Choices
+
+Our choices have evolved over time, not only as we have gained experience
+but also as little A has grown and had different requirements.
+
+## Diaper Sizes
+I chose to invest in a small stash of newborn cloth diapers because I wanted
```

Comment:
https://dictionary.cambridge.org/dictionary/english/stash

```suggestion
I chose to invest in a small set of newborn cloth diapers because I wanted
```

Yes, the word "stash" _can_ be used in this context, but "set" is simpler and easier for non-native english speakers. And easier to _translate_. 💭